examples/gno.land/p/demo/markdown/markdown.gno

```diff
@@ -0,0 +1,126 @@
+package markdown
+
+import (
+	"strings"
+
+	"gno.land/p/demo/ufmt"
+)
+
+// Bold returns bold text for markdown
+func Bold(text string) string {
+	return ufmt.Sprintf("**%s**", text)
+}
+
+// Italic returns italicized text for markdown
+func Italic(text string) string {
+	return ufmt.Sprintf("*%s*", text)
+}
+
+// Strikethrough returns strikethrough text for markdown
+func Strikethrough(text string) string {
+	return ufmt.Sprintf("~~%s~~", text)
+}
+
+// H1 returns a level 1 header for markdown
+func H1(text string) string {
+	return ufmt.Sprintf("# %s", text)
+}
+
+// H2 returns a level 2 header for markdown
+func H2(text string) string {
+	return ufmt.Sprintf("## %s", text)
+}
+
+// H3 returns a level 3 header for markdown
+func H3(text string) string {
+	return ufmt.Sprintf("### %s", text)
+}
+
+// BulletList returns an bullet list for markdown
+func BulletList(items []string) string {
+	var sb strings.Builder
+	for _, item := range items {
+		sb.WriteString(ufmt.Sprintf("- %s\n", item))
+	}
+	return sb.String()
+}
+
+// OrderedList returns an ordered list for markdown
+func OrderedList(items []string) string {
+	var sb strings.Builder
+	for i, item := range items {
+		sb.WriteString(ufmt.Sprintf("%d. %s\n", i+1, item))
+	}
+	return sb.String()
+}
+
+// TodoItem represents a single todo item
+type TodoItem struct {
+	Item string
+	Done bool
+}
+
+// TodoList returns a list of todo items with checkboxes for markdown
+func TodoList(items []TodoItem) string {
+	var sb strings.Builder
+
+	for _, item := range items {
+		checkbox := " "
+		if item.Done {
+			checkbox = "x"
+		}
+		sb.WriteString(ufmt.Sprintf("- [%s] %s\n", checkbox, item.Item))
+	}
+	return sb.String()
+}
+
+// Blockquote returns a blockquote for markdown
+func Blockquote(text string) string {
+	lines := strings.Split(text, "\n")
+	var sb strings.Builder
+	for _, line := range lines {
+		sb.WriteString(ufmt.Sprintf("> %s\n", line))
+	}
+	return sb.String()
+}
+
+// InlineCode returns inline code for markdown
+func InlineCode(code string) string {
+	return ufmt.Sprintf("`%s`", code)
+}
+
+// CodeBlock returns a code block for markdown, optionally specifying the language
+func CodeBlock(code string, language string) string {
```

Comment:
Maybe we could create individual helpers for `CodeBlock` and `LanguageCodeBlock`. We don't need to adhere strictly to markdown specifications. Instead, we should aim for a good compromise regarding helper names and parameters that will facilitate effective documentation generation and autocompletion. 

The current approach might be the best, but a more usage-based method could also be worth considering.

Reply:
8674cf017994ccfa0b1b652c41a12b0a577bf508